modules/targetAcquisition/pylonAcquisition.py

```diff
@@ -0,0 +1,96 @@
+import tensorflow as tf
+import numpy as np
+import multiprocessing as mp
+import cv2
+from pylonDetection.detect import Detection
+from time import sleep
+
+from tensorflow.compat.v1 import ConfigProto
+from tensorflow.compat.v1 import InteractiveSession
+
+config = ConfigProto()
+config.gpu_options.allow_growth = True
+session = InteractiveSession(config=config)
+
+class PylonAcquisition:
```

Comment:
Only nitpick, please rename this class and the file to targetAcquisition just to keep it all the same.

Reply:
should I just delete the old targetAcquisition and replace it with this one?

Reply:
Yes